info/guide/guide-TOC.md

```diff
@@ -1,3 +1,4 @@
+# Diktat Coding Style Guide, v.0.0.1
```

Comment:
How will we track these versions?

Reply:
I actually think that it is enough to have the version here